README.rst

```diff
@@ -1,12 +1,114 @@
+===========
+SaltCellar
+===========
+
+|license| |Build Status| |codecov| |Updates| |Python 3| |Docs|
+
+~~~~~
+About
+~~~~~
+
+SaltCellar's goal is to provide an open source solution for cost management of cloud and hybrid cloud environments. This is offered via a web interface that exposes resource consumption and cost data in easily digestible and filterable views. The project also aims to provide insight into this data and ultimately provide suggested optimizations for reducing cost and eliminating unnecessary resource usage.
+
+Full documentation is available through readthedocs_.
+
+
 Getting Started
----------------
+===============
 
-To deploy a new app on OpenShift Container Platform:
+This is a Python project developed using Python 3.6. Make sure you have at least this version installed.
 
-::
+Development
+------------
+To get started developing against SaltCellar first clone a local copy of the git repository. ::
 
     git clone https://github.com/SaltCellar/SaltCellar
-    cd SaltCellar
-    oc new-app openshift/templates/django-postgresql-persistent.json --code=https://github.com/SaltCellar/SaltCellar
 
+Developing inside a virtual environment is recommended. A Pipfile is provided. Pipenv is recommended for combining virtual environment (virtualenv) and dependency management (pip). To install pipenv, use pip ::
+
+    pip3 install pipenv
+
+Then project dependencies and a virtual environment can be created using ::
+
+    pipenv install --dev
+
+To activate the virtual environment run ::
+
+    pipenv shell
+
+Configuration
+~~~~~~~~~~~~~
+
+This project is developed using the Django web framework. Many configuration settings can be read in from a `.env` file. An example file `.env.example` is provided in the repository. To use the defaults simply ::
+
+    mv .env.example .env
+
+
+Modify as you see fit.
+
+Database
+~~~~~~~~
+
+PostgreSQL is used as the database backend for SaltCellar. A docker-compose file is provided for creating a local database container. If modifications were made to the .env file the docker-compose file will need to be modified to ensure matching database credentials. Several commands are available for interacting with the database. ::
```

Comment:
Could mention/link to running postgres locally? Can mention that non-container users can just call `make run-migrations`.